recipes/impyla/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: impyla
+  version: "0.13.6"
+
+source:
+  git_url: https://github.com/cloudera/impyla
+  git_rev: v0.13.6
```

Comment:
Again, could we please use the [PyPI release]( https://pypi.python.org/pypi/impyla/0.13.6 )?